src/SDKs/ServiceBus/ServiceBus.Tests/Tests/ScenarioTests.NamespaceTests.Detail.CRUD.cs

```diff
@@ -0,0 +1,183 @@
+        {
+            using (MockContext context = MockContext.Start(this.GetType().FullName))
+            {
+                InitializeClients(context);
+
+                var location = this.ResourceManagementClient.GetLocationFromProvider();
+
+                var resourceGroup = "v-ajnavtest";
+                //var resourceGroup = this.ResourceManagementClient.TryGetResourceGroup(location);
+                //if (string.IsNullOrWhiteSpace(resourceGroup))
+                //{
+                //    resourceGroup = TestUtilities.GenerateName(ServiceBusManagementHelper.ResourceGroupPrefix);
+                //    this.ResourceManagementClient.TryRegisterResourceGroup(location, resourceGroup);
+                //}
+
+                // Create Invalid Namespace
+                var invalidNamespaceName = TestUtilities.GenerateName(null);
+
+                //Check Invalid namespace name available
+                var checkInvalidNamespaceavailable = ServiceBusManagementClient.Namespaces.CheckNameAvailabilityMethod(new CheckNameAvailability() { Name = invalidNamespaceName });
+
+                Assert.Equal("InvalidName", checkInvalidNamespaceavailable.Reason.ToString());
+
+                // Create Namespace
+                var namespaceName = TestUtilities.GenerateName(ServiceBusManagementHelper.NamespacePrefix);
+
+                //Check namespace name available
+                var checknamespaceavailable = ServiceBusManagementClient.Namespaces.CheckNameAvailabilityMethod(new CheckNameAvailability() { Name = namespaceName });
+
+                var createNamespaceResponse = this.ServiceBusManagementClient.Namespaces.CreateOrUpdate(resourceGroup, namespaceName,
+                    new SBNamespace()
+                    {
+                        Location = location,
+                        Sku = new SBSku
+                        {
+                            Name = SkuName.Standard,
+                            Tier = SkuTier.Standard
+                        },
+                        Tags = new Dictionary<string, string>()
+                        {
+                            {"tag1", "value1"},
+                            {"tag2", "value2"}
+                        }
+                    });
+
+                Assert.NotNull(createNamespaceResponse);
+                Assert.Equal(createNamespaceResponse.Name, namespaceName);
+
+                TestUtilities.Wait(TimeSpan.FromSeconds(5));
+
+                // Get the created namespace
+                var getNamespaceResponse = ServiceBusManagementClient.Namespaces.Get(resourceGroup, namespaceName);
+                if (string.Compare(getNamespaceResponse.ProvisioningState, "Succeeded", true) != 0)
+                    TestUtilities.Wait(TimeSpan.FromSeconds(5));
+
+                getNamespaceResponse = ServiceBusManagementClient.Namespaces.Get(resourceGroup, namespaceName);
+                Assert.NotNull(getNamespaceResponse);
```

Comment:
please verify all properties are as we have set while creating the namespace

Reply:
Done!